tests/java/test1/TestFuzzer.java

```diff
@@ -0,0 +1,23 @@
+// Copyright 2022 Google LLC
```

Comment:
Fix copyright for fuzz introspector authors

Reply:
Fixed